src/app/entities/courses/directives/availability-status/availability-status.directive.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { AvailabilityStatusDirective } from './availability-status.directive';
+
+const CURRENT_DATE = new Date();
+const ONE_DAY_IN_MILLISECONDS = 1000 * 60 * 60 * 24;
+
+const OLD_DATE = new Date(CURRENT_DATE.getTime() - ONE_DAY_IN_MILLISECONDS * 30);
+const RECENT_DATE = new Date(CURRENT_DATE.getTime() - ONE_DAY_IN_MILLISECONDS * 7);
+const FUTURE_DATE = new Date(CURRENT_DATE.getTime() + ONE_DAY_IN_MILLISECONDS * 7);
+
+const COLORS = {
+  BLUE: '#009ECC',
+  GREEN: '#67A300',
+  TRANSPARENT: 'transparent',
+}
```

Comment:
COLORS object must be declared and exported in one place.

Reply:
Will do